nx/lib/nx.ex

```diff
@@ -9167,6 +9167,84 @@ defmodule Nx do
     end
   end
 
+  @doc """
+  Finds the variance of a tensor.
+
+  ## Examples
+
+  iex> Nx.var(Nx.tensor([[1, 2], [3, 4]]))
+
+  #Nx.Tensor<
+  f32
+  1.25
+  >
```

Comment:
```suggestion
      iex> Nx.var(Nx.tensor([[1, 2], [3, 4]]))
      #Nx.Tensor<
        f32
        1.25
      >
```

Please fix the other ones accordingly. :)

Reply:
sure.